### PR TITLE
 Use 1.0.0 dependencies

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Not merging this yet, just to illustrate what is changed when reverting to 1.0.0 dependencies.

### CODE REVIEW

File: samples/ResponseCachingSample/project.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-  "version": "1.1.0-*",
+  "version": "1.0.0-*",
   "dependencies": {
     "Microsoft.AspNetCore.ResponseCaching": "1.0.0-*",
-    "Microsoft.AspNetCore.Server.IISIntegration": "1.1.0-*",
-    "Microsoft.AspNetCore.Server.Kestrel": "1.1.0-*",
-    "Microsoft.Extensions.Caching.Memory": "1.1.0-*"
+    "Microsoft.AspNetCore.Server.IISIntegration": "1.0.0-*",
+    "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*",
+    "Microsoft.Extensions.Caching.Memory": "1.0.0-*"
```
Comment (Tratcher) [inline samples/ResponseCachingSample/project.json]: remove the `*` for everything outside of this repo

**Comment by JunTaoLuo** [comment]
:up::date: cc @pranavkm 

I plan to push these changes to a `rel/1.0.0` branch soon

**Comment by pranavkm** [comment]
Cool. Just make sure you have the branch right when pushing 👍

**Comment by JunTaoLuo** [comment]
@pranavkm is that a :shipit:?

File: NuGet.Config [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
+    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-master/api/v3/index.json" />
```
Comment (pranavkm) [inline NuGet.Config]: Could you add a `<clear />` here? Or copy the file from https://github.com/aspnet/Hosting/blob/1.0.0/NuGet.config

**Comment by pranavkm** [comment]
:shipit:

**Comment by JunTaoLuo** [comment]
Oops merged to dev using the wrong strategy. Will revert the changes on dev.

**Pull request merged**